File: base-forge-data-dictionary/src/main/java/com/gewuyou/baseforge/dictionary/repository/DictionaryCategoryRepository.java
package com.gewuyou.baseforge.dictionary.repository;

import com.gewuyou.baseforge.dictionary.model.DictionaryCategory;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
import org.springframework.stereotype.Repository;

/**
 * 字典类别存储库
 *
 * @author gewuyou
 * @since 2025-01-17 11:56:43
 */
@Repository
public interface DictionaryCategoryRepository extends JpaRepository<DictionaryCategory, Long>, JpaSpecificationExecutor<DictionaryCategory> {
}

File: base-forge-data-dictionary/src/main/kotlin/com/gewuyou/baseforge/dictionary/repository/DictionaryCategoryRepository.kt
package com.gewuyou.baseforge.dictionary.repository

import com.gewuyou.baseforge.dictionary.model.DictionaryCategory
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor

/**
 * 字典类别存储库
 *
 * @author gewuyou
 * @since 2025-01-17 11:56:43
 */
interface DictionaryCategoryRepository : JpaRepository<DictionaryCategory, Long>,
    JpaSpecificationExecutor<DictionaryCategory>
